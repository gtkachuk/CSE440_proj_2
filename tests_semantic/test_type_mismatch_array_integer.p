program testTypeMismatchArrayInteger;

class testTypeMismatchArrayInteger

BEGIN

   VAR 
      compilerWorks : boolean;
      timeToWrite   : ARRAY[4..9] OF integer;

FUNCTION testTypeMismatchArrayInteger;
BEGIN
   compilerWorks := timeToWrite[4]
END

END
.

