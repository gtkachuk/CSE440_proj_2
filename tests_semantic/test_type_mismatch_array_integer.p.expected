ERROR:14:Type mismatch between "boolean" and "integer"
Errors detected. Exiting.
